src/org/pentaho/reporting/platform/plugin/async/PentahoAsyncReportExecution.java

```diff
@@ -18,13 +19,13 @@
 
 public class PentahoAsyncReportExecution implements IAsyncReportExecution<InputStream> {
 
-  private final SimpleReportingComponent reportComponent;
-  private final AsyncJobFileStagingHandler handler;
-  private final String url;
-  private final String auditId;
-  private final IPentahoSession safeSession;
+  protected final SimpleReportingComponent reportComponent;
+  protected final AsyncJobFileStagingHandler handler;
+  protected final String url;
+  protected final String auditId;
+  protected final IPentahoSession safeSession;
 
```

Comment:
There is no need to set these variables as protected. Good encapsulation rules say that all data should be as invisible as possible. In the scope of this bug fix, there is no reason for this change.